src/mobile-routes/characters/character-details.tsx

```diff
@@ -1,14 +1,14 @@
 ﻿import React, { useMemo, useState } from 'react';
-import { ICharacter2, IMaterialRecipeIngredientFull } from '../../models/interfaces';
+import { ICharacter2, IMaterialRecipeIngredientFull } from 'src/models/interfaces';
```

Comment:
This was an IDE suggestion. It worked when tested locally. I can revert it if desired

Reply:
it's fine